tooling/template-base/components/LearnerAccess/LoadCertificates.tsx

```diff
@@ -11,14 +11,47 @@ import {
 import useLearnerAccess from './Context/use-context';
 import { t } from 'i18next';
 import LearnerAccessGridView from './Views/GridView';
+import { usePageContext } from '../../renderer/usePageContext';
+import clsx from 'clsx';
 
 type RequiredUserCertifacesQuery = Required<UserCertificatesQuery>;
 type UserCertificate = RequiredUserCertifacesQuery['UserCertificates'][0];
 interface CertificateProps {
   item: UserCertificate;
 }
 const Certificate = ({ item }: CertificateProps) => {
-  return <LearnerAccessGridView item={item.contentItem} />;
+  const contentItem = item.contentItem;
+  const { appearance } = usePageContext();
+  const assetImage = contentItem?.asset ? contentItem.asset : appearance?.logoAsset;
+  return (
```

Comment:
What was the thought for swapping this in instead of `LearnerAccessGridView`? Wasn't sure if something was mistakenly swapped or if it was intentional

Reply:
types got a bit complicated. Found that it would have been a bit of a mess, especially if the user wanted to edit the components in any way.

Reply:
@AlexONeillTI Can you expand on that? Unsure how types here would differ from other areas where we're calling `LearnerAccessGridView`

Reply:
So think this can be resolved by adjusting our type definitions to something like:

```typescript
type UserCertificateItem = {
  contentItem: HydratedContentItem;
};

interface CertificateProps {
  item: UserCertificateItem;
}
```

Then you should be able to `return <LearnerAccessGridView item={item.contentItem} />;` without issues.

Reply:
error when calling `Certificate`:
`{data?.UserCertificates?.map(item => (
     <Certificate key={item.id} item={item} />
  ))}` 
  returns a type error `Certificate` type and `UserCertificateItem` are not compatible. Will try and fix this now

Reply:
Ah gotcha. Can probably leave types as you have them defined, then do something like this:

```jsx
const { i18n } = useTranslation();
if (item.contentItem) {
  const contentItem = item.contentItem as Content;
  const hydratedContentItem = hydrateContent(i18n, contentItem);
  return <LearnerAccessGridView item={hydratedContentItem} />;
}
```

Reply:
done and pushed